tests/data/sampler/largestTriangle3Spec.js

```diff
@@ -0,0 +1,62 @@
+describe('fc.data.sampler.largestTriangle3', function() {
+
+    var data;
+    var dataGenerator;
+
+    describe('simple xy data', function() {
+
+        beforeEach(function() {
+
+            data = [
+                { x: 0, y: 6 }, // Bucket 1
```

Comment:
How did you create this test data? Would it be possible to use someone else's implementation as the basis for your test?

https://github.com/pingec/downsample-lttb